boards/mediatek/mt8188/mt8188_adsp.dts

```diff
@@ -0,0 +1,81 @@
+/* Copyright 2024 The ChromiumOS Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <mem.h>
+
+/dts-v1/;
+/ {
+
+	#address-cells = <1>;
+	#size-cells = <1>;
+
+	sram0: memory@4e100000 {
+		 device_type = "memory";
+		 compatible = "mmio-sram";
+		 reg = <0x4e100000 DT_SIZE_K(512)>;
+	};
+
+	dram0: memory@60000000 {
+		device_type = "memory";
+		compatible = "mmio-sram";
+		reg = <0x60000000 DT_SIZE_M(15)>;
+	};
+
+	dram1: memory@61000000 {
+		device_type = "memory";
+		compatible = "mmio-sram";
+		reg = <0x61000000 DT_SIZE_K(1024)>;
+	};
+
+	soc {
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		core_intc: core_intc@0 {
+			compatible = "cdns,xtensa-core-intc";
+			reg = <0 4>;
+			interrupt-controller;
+			#interrupt-cells = <3>;
+		};
+
+		intc2: intc@10b80010 {
+			compatible = "mediatek,adsp_intc";
+			interrupt-controller;
+			#interrupt-cells = <3>;
+			reg = <0x10b80010 4>;
+			status-reg = <0x10b80050>;
+			interrupts = <2 0 0>;
+			mask = <0x3f>;
+			interrupt-parent = <&core_intc>;
+		};
+
+		ostimer64: ostimer64@10b83080 {
+			compatible = "mediatek,ostimer64";
+			reg = <0x10b83080 28>;
+		};
+
+		ostimer0: ostimer@10b83000 {
+			compatible = "mediatek,ostimer";
+			reg = <0x10b83000 16>;
+			interrupt-parent = <&core_intc>;
+			interrupts = <18 0 0>;
+		};
```

Comment:
nit: could/should these entries be in reversed order, given their addresses?

Reply:
They're not the same kind of device.  The first is the 13MHz 64 bit upcounter used for system timing, the second is a 32 bit 26MHz down-counting interrupt source.  The timer driver uses them both, but in fact the second has multiple instantiations on the chips (though not used by Zephyr), so they make sense to declare separately.  As far as order: my sense of aesthetics likes having the master counter first, I guess.